pom.xml

```diff
@@ -306,119 +103,6 @@
 
         </pluginManagement>
 
-        <plugins>
-
-            <plugin>
-                <groupId>org.apache.maven.plugins</groupId>
-                <artifactId>maven-enforcer-plugin</artifactId>
-                <version>${maven-enforcer-plugin.version}</version>
-                <executions>
-                    <execution>
-                        <id>enforce</id>
-                        <configuration>
-                            <rules>
-                                <dependencyConvergence />
-                                <bannedDependencies>
-                                    <excludes>
-                                        <!-- As recommended from slf4j guide, exclude commons-logging -->
```

Comment:
Does kiwi-parent have these? I don't think it does, so why removing?

Reply:
Or, do we need to _add_ them to kiwi-parent?

Reply:
Well I guess that is a good question.  Why aren't we banning those?

Reply:
So, actually in our base parent POM for services, we _do_ ban them. I am not sure why we didn't in kiwi.

Reply:
I'm going to peruse the SLF4J guide since that first comment mentions it.

Reply:
I don't see anything in the SLF4J docs about this, so maybe it used to be there years ago but now is not?

Reply:
For now, let's leave this alone, but let's create a new discussion or investigation issue to see whether we should add these.

I think maybe kiwi doesn't have it, because while we use the SLF4J _API_ in our code, we don't enforce a specific logging back-end, so you could use Logback, Log4j2, JUL, etc.

But in our Dropwizard services, we enforce those bans because Dropwizard uses Logback, and thus we definitely do _not_ want any SLF4J bindings to other back-ends, as they could potentially cause fun things like infinite delegation loops (e.g. see [Delegation Loop](https://www.slf4j.org/codes.html#jclDelegationLoop)) - at least until you get a `StackOverflowError`...

Reply:
Created #32